src/main/java/com/configcat/ConfigCatLogger.java

```diff
@@ -2,54 +2,62 @@
 
 import org.slf4j.Logger;
 
+import java.util.function.Function;
+
 class ConfigCatLogger {
     private final Logger logger;
     private final LogLevel logLevel;
     private final ConfigCatHooks configCatHooks;
+    private final Function<FilterFunctionParameters, Boolean> filterOutFunction;
 
-    public ConfigCatLogger(Logger logger, LogLevel logLevel, ConfigCatHooks configCatHooks) {
+    public ConfigCatLogger(Logger logger, LogLevel logLevel, ConfigCatHooks configCatHooks, Function<FilterFunctionParameters, Boolean> filterOutFunction) {
         this.logger = logger;
         this.logLevel = logLevel;
         this.configCatHooks = configCatHooks;
+        this.filterOutFunction = filterOutFunction;
```

Comment:
nit: Similar stuff, I think `filterFunction` would be sufficuent.

```suggestion
    public ConfigCatLogger(Logger logger, LogLevel logLevel, ConfigCatHooks configCatHooks, Function<FilterFunctionParameters, Boolean> filterFunction) {
        this.logger = logger;
        this.logLevel = logLevel;
        this.configCatHooks = configCatHooks;
        this.filterFunction = filterFunction;
```